client/src/app/containers/Home/ActiveGamesTable.jsx

```diff
@@ -29,81 +46,81 @@ const ActiveGamesTable = () => {
     dispatch(homeActions.refreshActiveGames({ id: currentUser.id }));
   };
 
-  let gameButtons = activeGames.map(activeGame => {
+  let createData = (gameID, startedTime, lastMoveTime, opponents, color) => {
+    let myColor = color
+      ? color.includes('red')
+        ? red[800]
+        : color.includes('blue')
+        ? blue[800]
+        : color.includes('green')
+        ? green[800]
+        : grey[400]
+      : grey[400];
+    return { gameID, startedTime, lastMoveTime, opponents, myColor };
+  };
```

Comment:
This entire function is passing through its args except for `color`. Maybe we don't 5 input args?